src/core/bitop.d

```diff
@@ -72,20 +72,17 @@ unittest
  */
 int bsf(size_t v) pure;
 
+version(CoreDdoc)
 /// ditto
+int bsf(ulong v) pure;
+
+version(D_LP64) { } else
```

Comment:
> It's silly to not use language constructs just because someone can be uninformed about it.

I already explained why I used `static if`. Also, some of the similar uses of `static if` are from **Walter Bright** himself.

Just because *you* wouldn't do it the way I did, doesn't mean I'm "uninformed".

> Yes, but I will not use the style I'm against of.

Then fix the whole file, if you think it's that important.